src/externalized/DefaultContentManager.cc

```diff
@@ -482,13 +476,8 @@ bool DefaultContentManager::doesGameResExists(char const* filename) const
 			file = fopen(path, "rb");
 			if (!file)
 			{
-				LibraryFile* libFile = LibraryFile_open(m_libraryDB, filename);
-				if (libFile)
-				{
-					LibraryFile_close(libFile);
-					return true;
-				}
-				return false;
+				RustPointer<LibraryFile> libFile(LibraryFile_open(m_libraryDB.get(), filename));
+				return static_cast<bool>(libFile);
```

Comment:
Should it be `libFile.get()` here?

Reply:
`std::unique_ptr` has a bool operator, I'm using it by casting to bool.

Reply:
:+1: